docker/setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+git clone --branch HC-158_lightweight-jobs https://github.com/hysds/hysds_commons.git
```

Comment:
@DustinKLo: does this need to be `--branch develop-es7`? Could you create a TODO comment here so that we can easily search for it when we eventually merge in develop-es7 to develop.

Reply:
@pymonger hysds_commons isn't yet merged to develop-es7 so this was the only way to test my lightweight-jobs
will change after changes to hysds_commons gets approved

Reply:
ok